### DISCUSSION TITLE
Welcome to hms-mqtt-publisher Discussions!

### DISCUSSION THREAD

**Discussion by DennisOSRM** [DennisOSRM/hms-mqtt-publisher#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smiley561** [comment]
Hi DennisOSRM,

do you have any documentation of how to use it?

Many Greetings Bernd

**Reply by DennisOSRM** [reply]
The readme should get you started: https://github.com/DennisOSRM/hms-mqtt-publisher/blob/main/README.md
